docs/run.js

```diff
@@ -100,9 +100,12 @@ function createAndDisplayJoinedData(map, initialTogglesValuesPromise) {
   // clear old listeners
   google.maps.event.removeListener(mapSelectListener);
   google.maps.event.removeListener(featureSelectListener);
+  const dataPromise = resolvedScoreAsset.then(getEePromiseForFeatureCollection);
+  dataPromise.catch(
```

Comment:
can you attach this to the above line?

Reply:
I don't think so: we want the functions that are called below with dataPromise to see the thrown error, so they don't try to do anything. If we passed the `dataPromise.catch` expression into them, they'd see a successful promise (although one with a value of `undefined`).

If we threw inside the `.catch` block that would prevent them seeing an unsuccessful promise, but I'm not sure it gains anything. I can do that, if you prefer? Will go ahead and submit this after responding to other reviews, but can change in a follow-up.